wgpu-types/src/lib.rs

```diff
@@ -593,7 +593,15 @@ bitflags::bitflags! {
         /// - All
         ///
         /// This is a native only feature.
-        const CLEAR_COMMANDS = 0x0000_0001_0000_0000;
+        const CLEAR_COMMANDS = 0x0000_0200_0000_0000;
+        /// Enables creating shader modules from SPIR-V binary data (unsafe).
+        ///
+        /// Supported platforms:
+        /// - Vulkan, in case shader's requested capabilities and extensions agree with
+        /// Vulkan implementation.
+        ///
+        /// This is a native only feature.
+        const SPIR_V_SHADER_MODULES = 0x0000_0400_0000_0000;
```

Comment:
can we not put `V` aside? maybe just `SPIRV_SHADERS`?

Reply:
Oh yeah, that's probably a good idea.